sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1186,6 +1186,80 @@ def max_deflection(self):
         else:
             return None
 
+
+    def bending_stress(self):
+        """
+        Returns bending stress of a beam cross-section at a distance `y` from the neutral axis.
+
+        Example
+        =======
+        There is a beam of length 6 m with a u.d.l of 40 KN/m over its length. The beam has a rectangular
+        cross-section(0.3 x 0.6 m) with moment of inertia. The bending stress at a point
+        0.2 m above the neutral axis and at a cross-section which is 4 meters from the start of the beam
+        has to be determined.
+
+        >>> from sympy.physics.continuum_mechanics.beam import Beam
+        >>> from sympy import symbols, Piecewise
+        >>> E, I = symbols('E, I')
+        >>> R1, R2 = symbols('R1, R2')
+        >>> b = Beam(6, E, I)
+        >>> b.apply_load(R1, 0, -1)
+        >>> b.apply_load(40, 0, 0, end=6)
+        >>> b.apply_load(R2, 6, -1)
+        >>> b.solve_for_reaction_loads(R1, R2)
+        >>> b.bending_stress()
+        y*(-120*SingularityFunction(x, 0, 1) + 20*SingularityFunction(x, 0, 2)
+        - 120*SingularityFunction(x, 6, 1) - 20*SingularityFunction(x, 6, 2))/I
+
+        """
+        y = Symbol('y')
+        M = self.bending_moment()
+        b_stress = (M*y)/self.second_moment
+        return b_stress
+
+
+
+    def shear_stress(self):
+        """
+        Returns the shear stress of a beam cross-section at point(or layer) above the
+        neutral axis with its first moment of area as `Q` and width of the cross-section as 't'.
+
+        Example
+        =======
+        A cantilever beam of I-cross-section and length 8 meters is under downward distributed constant load with magnitude
+        of 5.0 KN/m from starting point till 2 meters away from it. A ramp load of 2 kN/m applied from the mid till the end of the beam.
+        A point load of 12KN is also applied in same direction 4 meters away from start.
+
+        >>> from sympy.physics.continuum_mechanics.beam import Beam
+        >>> from sympy import symbols, Symbol
+        >>> E,I,M,V = symbols('E I M V')
+        >>> b = Beam(8, E, I)
+        >>> b.apply_load(V, 0, -1)
+        >>> b.apply_load(M, 0, -2)
+        >>> b.apply_load(5, 0, 0, end=2)
+        >>> b.apply_load(12, 4, -1)
+        >>> b.apply_load(2, 3, 1, end=8)
+        >>> b.solve_for_reaction_loads(V, M)
+        >>> tau = b.shear_stress()
+        >>> tau
```

Comment:
This routine should return `tau/(Q/I/t)` -- let the user multiply by `Q/It`. Is `tau/(Q/I/t)` shear strain?